data.sql

```diff
@@ -8,4 +8,12 @@ NSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg) V
 INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES ('Squirtle', '1993-04-02', 3, FALSE, -12.13);
 INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES ('Plantmon', '2021-11-15', 2, TRUE, -5.7);
 INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES ('Angemon', '2005-06-12', 1, TRUE, -45);
-INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES ('Boarmon', '2005-06-07', 7, TRUE, 20.4), ('Blossom', '1998-10-13', 3, TRUE, 17), ('Ditto', '2022-05-14', 4, TRUE, 22);
\ No newline at end of file
+INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES ('Boarmon', '2005-06-07', 7, TRUE, 20.4), ('Blossom', '1998-10-13', 3, TRUE, 17), ('Ditto', '2022-05-14', 4, TRUE, 22);
+INSERT INTO owners(full_name, age) VALUES ('Sam Smith', 34), ('Jennifer Orwell', 19), ('Bob', 45), ('Melody Pond', 77), ('Dean Winchester', 14), ('Jodie Whittaker', 38);
```

Comment:
Please insert `Pokemon` and `Digimon` into the `species` table.

Reply:
Hi @KDlamini 
Thanks for your review, I totally forgot to add that insert to the data file🙃
Happy coding!